TIUIKitCore/README.md

```diff
@@ -4,11 +4,87 @@ Core UI elements: protocols, views and helpers.
 
 # Protocols
 
-- [InitializableView](InitializableView/InitializableView.swift) - protocol with methods that should be called in constructor methods of view.
-- [Animatable](Animatable/Animatable.swift) - protocol that ensures that specific type support basic animation actions.
-- [ActivityIndicator](ActivityIndicator/ActivityIndicator.swift) - basic activity indicator.
-- [ActivityIndicatorHolder](ActivityIndicator/ActivityIndicatorHolder.swift) - placeholder view, containing activity indicator.
+- [InitializableView](Sources/InitializableView/InitializableViewProtocol.swift) - protocol with methods that should be called in constructor methods of view.
+- [Animatable](Sources/ActivityIndicator/Animatable.swift) - protocol that ensures that specific type support basic animation actions.
+- [ActivityIndicator](Sources/ActivityIndicator/ActivityIndicator.swift) - basic activity indicator.
+- [ActivityIndicatorHolder](Sources/ActivityIndicator/ActivityIndicatorHolder.swift) - placeholder view, containing activity indicator.
+- [AlertLocalizationProvider](Sources/Localization/AlertsLocalization/AlertLocalizationProvider.swift) - protocol that ensures that localization for alerts will be provided.
+- [AlertPresentable](Sources/Alerts/Protocols/AlertPresentable.swift) - protocol indicates that certain object can present alerts.
+- [AlertPresentationContext](Sources/Alerts/Protocols/AlertPresentationContext.swift) - protocol indicates that certain object can present alert on top of itself.
+- [UIKitAlertContext](Sources/Alerts/Protocols/UIKitAlertContext.swift) - helper to provide easy conformance of `UIViewController` to `AlertPresentationContext`.
 
-# Views
+# Models
 
-- [BaseInitializableView](BaseInitializableView/BaseInitializableView.swift) - UIView conformance to InitializableView.
+- [DefaultAlertLocalizationProvider](Sources/Localization/AlertsLocalization/DefaultAlertLocalizationProvider.swift) - default localization provider for alerts.
+- [AlertAction](Sources/Alerts/Models/AlertAction.swift) - representation of alert action
+- [AlertDescriptor](Sources/Alerts/Models/AlertDescriptor.swift) - struct that holds all needed information to present alert
+
+# Factories
+
+- [AlertsFactory](Sources/Alerts/Factories/AlertFactory.swift) - factory to present alerts.
+
+## AlertsFactory
+Use to present alerts in a few lines of code. Can be used for UIKit and SwiftUI
+> You can initialize `AlertsFactory` with your own *LocalizationProvider* or use `DefaultAlertLocalizationProvider`
+
+### Your view or view controller must implement PresentationContext protocol
+There are `UIKitAlertContext` protocol that are designed to make it easier to work with `AlertPresentationContext` protocol. By default, no changes need to be made for UIKit view controllers to make them conform to `UIKitAlertContext`. 
```

Comment:
There is a `UIKitAlertContext` protocol designed to make it easier to work with `AlertPresentationContext` protocol.